src/pages/frontpage/NvaDescriptionSection.tsx

```diff
@@ -16,8 +19,16 @@ const iconStyle = { height: '2rem', width: '2rem' };
 export const NvaDescriptionSection = () => {
   const { t } = useTranslation();
 
+  const registrationQuery = useRegistrationSearch({
+    params: { aggregation: 'all' },
+  });
+
+  const personQuery = useSearchForPerson({});
```

Comment:
Siden personsøk har en raritet som gjør at man ikke kan be om 0 treff
```suggestion
  const personQuery = useSearchForPerson({ results: 1 });
```

Reply:
Nice to know 🤔 Burde fikses på backend?